docs/tutorial/Basic/Setting Up.mdx

```diff
@@ -75,10 +76,45 @@ Change directory into the repository and install the project dependencies
 
 Alternative set up paths can be just cloning or forking the example repository, or starting from scratch using our [fhenix hardhat plugin](../../devdocs/docs/hardhat), [Gitpod environment](../../devdocs/docs/gitpod) or Github Workspaces
 
-## Starting Localfhenix
+## Configuring and Starting Localfhenix
+In order for us to locally deploy and test contracts we will need to run our own node, for this specific use-case we have a docker image named `localfhenix`.
+For ease of use and a simple setup we have hardhat task `localfhenix:start` that downloads and runs the image in a container for you.
 
-npm run hardhat localfhenix:start
+#### Set up your Mnemonic
 
+Hardhat uses an `.env` with mnemonics to deploy and interact with contracts. Copy the `.env.example` file to `.env` and set your mnemonics (you can use the default mnemonics if you just want to use LocalFhenix)
+
+```bash
+cp .env.example .env
+```
+This is the output of `cat .env`:
+```bash
+# obviously DO NOT USE IN PRODUCTION OR ANYWHERE OR WHATEVER
+# default wallet of localfhenix
+export MNEMONIC="demand hotel mass rally sphere tiger measure sick spoon evoke fashion comfort"
+export WALLET=""
+```
+#### Starting LocalFhenix.
+
+Use the following command:
+
+<Tabs groupId="package-managers">
+    <TabItem value="npm" label="npm">
```

Comment:
Have you actually tried running these? I recall it used to not work with package managers other than pnpm for some reason, not sure if that's still the case or not

Reply:
@toml01 so I did try running them, just not on a properly clean repo. Everything seemed to work fine. Just as a sanity-check I might pull 3 different clones and test it out again.

Reply:
So I did test it today, and it does work. However I've looked into how the package managers work and I will probably add a Disclaimer on top saying that our projects mainly use (p)npm and that although yarn can be used it is not recommended. (Due to missing lock files)